cdblib/cdblib.py

```diff
@@ -193,19 +231,19 @@ def __exit__(self, exc_type, exc_value, traceback):
 
     def put(self, key, value=b''):
         '''Write a string key/value pair to the output file.'''
-        if (
-            (not isinstance(key, six.binary_type)) or
-            (not isinstance(value, six.binary_type))
-        ):
-            msg = 'key and value must be of type {}'
+        # Ensure that the value is binary
+        if not isinstance(value, six.binary_type):
```

Comment:
I could be persuaded to attempt to coerce values to binary in non-`strict` mode, but we already have `putint` and `putstring`.